operate/Makefile

```diff
@@ -5,12 +5,16 @@ env-up:
 	docker compose -f docker-compose.yml up -d elasticsearch \
     && mvn install -DskipTests=true -Dskip.fe.build=false -DskipChecks -DskipQaBuild \
 	&& CAMUNDA_OPERATE_IMPORTERENABLED=false \
+	CAMUNDA_SECURITY_AUTHORIZATIONS_ENABLED=false \
+	CAMUNDA_OPERATE_ELASTICSEARCH_CREATESCHEMA=false \
```

Comment:
🔧  I guess this also helped to avoid the exporter getting stuck? I wonder if we stick to this approach or also use `ZEEBE_BROKER_EXPORTERS_CAMUNDAEXPORTER_ARGS_INDEX_SHOULDWAITFORIMPORTERS=false` here, wdyt?

Reply:
Possible, I will switch it to the should wait for importers config though (along with the other places)